underactuated.html

```diff
@@ -5151,7 +5151,7 @@
   Again, this algorithm <em>is</em> probabilistically complete.  But after expanding 1000 nodes, the tree is basically a mess of node points all right on top of
   each other:
   <figure>
-    <img width="70%" src="figures/random_tree.svg"></img>
+    <img width="70%" src="figures/random_tree.svg">
```

Comment:
this one looks incorrect.  Although I do normally end a single tag with /> for img